tests/commons.py

```diff
@@ -21,10 +27,18 @@ async def __anext__(self):
         return item
 
     def __call__(self, *args, **kwargs):
+        self.call_count += 1
+
         if args:
             self.call_args.append(args)
 
         if kwargs:
             self.call_kwargs.append(kwargs)
 
         return self
+
+    def assert_not_called(self):
+        return self.call_count == 0
```

Comment:
It baffles me that this isn't part of pytest. I've run across https://stackoverflow.com/questions/54838354/python-how-can-i-assert-a-mock-object-was-not-called-with-specific-arguments before. Lots of ways to hack around it I guess.